st2common/st2common/rbac/resolvers.py

```diff
@@ -176,8 +217,37 @@ class SensorPermissionsResolver(PermissionsResolver):
     """
 
     def user_has_permission(self, user_db, permission_type):
-        # TODO
-        return True
+        assert permission_type in [PermissionType.SENSOR_LIST]
```

Comment:
There is a lot of code duplication here. Can you make a note in JIRA to refactor this?